resources/recipes.py

```diff
@@ -416,6 +428,9 @@ def plank(thing: str):
     alloy_recipe(rm, 'weak_blue_steel', 'weak_blue_steel', ('black_steel', 0.5, 0.55), ('steel', 0.2, 0.25), ('bismuth_bronze', 0.1, 0.15), ('sterling_silver', 0.1, 0.15))
     alloy_recipe(rm, 'weak_red_steel', 'weak_red_steel', ('black_steel', 0.5, 0.55), ('steel', 0.2, 0.25), ('brass', 0.1, 0.15), ('rose_gold', 0.1, 0.15))
 
+def disable_recipe(rm: ResourceManager, name_parts: utils.ResourceIdentifier):
```

Comment:
We shouldn't need a custom recipe for this (and overriding with an explicit "noop" is clutter that should be avoided, since it pollutes the recipe lists). Does [this recipe removal](https://github.com/alcatrazEscapee/no-tree-punching/blob/1.18.x/src/main/resources/data/minecraft/recipes/acacia_planks.json) not work in 1.18?